lib/dockerCompose.ts

```diff
@@ -0,0 +1,111 @@
+import { ActiveDeployment } from './deployment';
+import { RA } from './typescriptCommonTypes';
+
+export const createDockerConfig = (
+  deployments: RA<ActiveDeployment>,
+  nginxConfigHash: number
+): string => `
+version: '3.9'
+services:
+
+${
+  deployments.length === 0
+    ? `
+  mariadb:
+    restart: unless-stopped`
+    : ''
+}
+${deployments
+  .map(
+    (deployment) => `
+  ${deployment.hostname}:
+    image: specifyconsortium/specify7-service:${deployment.branch}
+    init: true
+    restart: unless-stopped
+    networks:
+      - database
+      - nginx
```

Comment:
Pretty sure the `redis` network will be needed here.

Reply:
Fixed in 060bbd9